### PR TITLE
Release 7.1.0

### PR DESCRIPTION
**Pull request by manicki**
Updated 7.x branch. I hope I did not miss any non-breaking change since 7.0.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Wikibase DataModel release notes
 
+## Version 7.1.0 (2017-08-30)
+
+* `EntityIdValue::getArrayValue` can handle foreign entity IDs and entity IDs that do not have a numeric representation
```
Comment (JeroenDeDauw) [inline RELEASE-NOTES.md]: This is really minor and you can be changed whenever: this sentence does not follow the usual format/grammar of release notes. It does not explicitly say what changed

Comment (manicki) [reply RELEASE-NOTES.md]: true. rephrased. not sure it explains what changed well enough though.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Wikibase DataModel release notes
 
+## Version 7.1.0 (2017-08-30)
```
Comment (lucaswerkmeister) [inline RELEASE-NOTES.md]: date is now out-of-date :/

Comment (manicki) [reply RELEASE-NOTES.md]: Aight. Bumped to sugar-free september.

**Comment by JeroenDeDauw** [comment]
I saw 7.1.0 and then realized this is not a PHP version :(

**Comment by JeroenDeDauw** [comment]
Not going to tag this myself since I am not sure what is going on with having a 7.x branch while master also appears to be at 7.x.

**Comment by manicki** [comment]
Master is already 8.x. Let me fix master and then tag 7.1.0.

**Pull request merged**